scripts/provision.sh

```diff
@@ -200,7 +200,7 @@ start_datastore() {
 }
 
 start_minio() {
-  sudo hab svc load "${BLDR_ORIGIN}/builder-minio" --channel "${BLDR_CHANNEL}" --force
+  sudo hab svc load "${BLDR_ORIGIN}/builder-minio" --channel "${BLDR_MINIO_CHANNEL}" --force
```

Comment:
Can we have `BLDR_MINIO_CHANNEL` fall back to `BLDR_CHANNEL` if not set? Otherwise we could break others installing with this script.